src/components/OfflineSearch.ts

```diff
@@ -0,0 +1,173 @@
+import { Component, Props, createElement } from "react";
+import { findDOMNode } from "react-dom";
+
+import * as dijitRegistry from "dijit/registry";
+
+import { Alert } from "./Alert";
+import "../ui/OfflineSearch.css";
+
+type searchMethodOptions = "equals" | "lessThan" | "lessThanOrEquals" | "greaterThan" | "greaterThanOrEquals" | "contains";
```

Comment:
SearchMethodOptions...
Type definitions and interfaces should be in upper camel case/